### ISSUE TITLE
Spike: Medallia Technical Implementation

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8281]
AC:

- Document the steps to implement Medallia, based on the native Medallia documentation [here](https://docs-assets.medallia.com/digital/docs-v2/mobile-sdk-docs/index.html#pages/getting-started/quick-start/quick-start-reactnative/getting-started-reactnative.html)
- Timeboxed to a half day

**Comment by timwright12** [comment]
Where are we on this one? I see it was moved to in progress 6 days ago (inclusive to the weekend), but the description says we're time boxing it to half a day

**Comment by cadibemma** [comment]
Ran into errors trying to implement on both android and ios. So after a discussion with Binny, we decided to hold off on this for now and we'll soon schedule a meeting for FE global to try and get it working.

**Comment by dumathane** [comment]
Lingering questions about how this would work AFTER we have a working Medallia SDK in the app (Which Theo is working on). I've attempted order of operations here:
1) Do we very easily get the API key required to authenticate with Medallia for free with VA? (if we need to buy something or can't easily even get into the Medallia software this is #1) I've requested access to the documentation via the how to guide in contact center for now.
2) How do we access the Medallia software for the VA pre-existing form library etc? Does this have to be through the 'contact center support' team (#vsp-contact-center-support) or via a PIV card access point on a remote pc?
3) Who will create our form in their proprietary software?
4) Is it easy to take a pre-existing web form and make it react-native accessible?
5) Once we have a working form, how do we see the data analytics for it's responses?
6) Does Medallia have any control baked in over who and how the form is presented or is that going to be all mobile app code?

I'll attach any more questions in addition as they come to me.
